website/static/js/filepage/revisions.js

```diff
@@ -132,7 +132,7 @@ var FileRevisionsTable = {
     view: function(ctrl) {
         return m('#revisionsPanel.panel.panel-default', [
                 m('.panel-heading.clearfix', m('h3.panel-title', 'Revisions')),
-                m('.panel-body', {style:{'padding-right': '0px','padding-left':'0px'}}, (function() {
```

Comment:
Css styles don't need Units if the value is 0. 



—
Sent from Mailbox

On Mon, Jul 13, 2015 at 10:16 AM, Steven Loria <notifications@github.com>
wrote:

>> @@ -132,7 +132,7 @@ var FileRevisionsTable = {
>>      view: function(ctrl) {
>>          return m('#revisionsPanel.panel.panel-default', [
>>                  m('.panel-heading.clearfix', m('h3.panel-title', 'Revisions')),
>> -                m('.panel-body', {style:{'padding-right': '0px','padding-left':'0px'}}, (function() {
> Why were the `px`'s removed here?
> From the [mithril docs](https://lhorie.github.io/mithril/mithril.html#usage):
>>  Mithril does not auto-append units like px or % to any values. 
> ---
> Reply to this email directly or view it on GitHub:
> https://github.com/CenterForOpenScience/osf.io/pull/3553/files#r34466644